python/ql/lib/semmle/python/frameworks/Stdlib.qll

```diff
@@ -4396,6 +4396,145 @@ private module StdlibPrivate {
       preservesValue = true
     }
   }
+
+  // ---------------------------------------------------------------------------
+  // asyncio
+  // ---------------------------------------------------------------------------
+  /** Gets a reference to the `asyncio` module. */
+  API::Node asyncio() { result = API::moduleImport("asyncio") }
```

Comment:
I totally understand why you ended up writing your code like you did, since the rest of the framework modeling is littered with stuff like this. It's from a dark time where we did not have API graphs. 

For new code, I would inline the `API::moduleImport("asyncio")` unless it was not possible.

```suggestion
```